sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/LeaseAction.java

```diff
@@ -4,50 +4,41 @@
 
 package com.azure.storage.file.datalake.models;
 
+import com.azure.core.util.ExpandableStringEnum;
 import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Collection;
 
 /** Defines values for LeaseAction. */
-public enum LeaseAction {
-    /** Enum value acquire. */
-    ACQUIRE("acquire"),
+public final class LeaseAction extends ExpandableStringEnum<LeaseAction> {
```

Comment:
I think we should leave this as an enum, same as it is on the other platforms.  It is very unlikely to ever expand.